pkg/model/proto/model.proto

```diff
@@ -717,6 +717,7 @@ message CreateFileShareOpts {
     // The locality that file share belongs to, required.
     string availabilityZone = 6;
     // The uuid of the pool on which file share will be created, required.
+    string profileId = 7;
```

Comment:
The profileId field is not required anymore, please remove it.

Reply:
Removed